docs/code-of-conduct.md

```diff
@@ -0,0 +1,127 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+**Hyperledger Governing Process document - do not modify without first getting approval of the Technical Steering Committee**
```

Comment:
We can probably strike this line. I believe it was so people wouldn't try to modify it on the wiki. All of the docs in this repo should be considered governing docs that require TSC change control.